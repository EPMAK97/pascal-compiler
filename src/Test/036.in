1.e+1+2