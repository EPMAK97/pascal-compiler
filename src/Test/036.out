1  |  1 | IDENTIFIER  | VARIABLE                    | g         | 
1  |  3 | OPERATOR    | OP_PLUS                     | +         | 
1  |  5 | IDENTIFIER  | VARIABLE                    | a         | 
1  |  7 | OPERATOR    | OP_MULT                     | *         | 
1  |  9 | IDENTIFIER  | VARIABLE                    | b         | 
1  | 11 | OPERATOR    | OP_MULT                     | *         | 
1  | 13 | IDENTIFIER  | VARIABLE                    | c         | 
1  | 15 | OPERATOR    | OP_PLUS                     | +         | 
1  | 17 | IDENTIFIER  | VARIABLE                    | m         | 
1  | 19 | OPERATOR    | OP_PLUS                     | +         | 
1  | 21 | IDENTIFIER  | VARIABLE                    | n         | 
END OF FILE
